### ISSUE TITLE
[Bug] TitleStyle uses serif font

### ISSUE THREAD

**Issue by breyed** [xamarin/Xamarin.Forms#7674]
On iOS, TitleStyle has switched from being a sans-serif font, as shown in [documentation](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/text/styles), to a serif font. I don't know if it was due updating to Xamarin.Forms 4.2.0.815419, to VS 16.3.0/3.8, Xcode 11, or to iOS 13.1. Whatever the cause, the new font doesn't fit with the other styles or the docs.

This is the documented sans-serif style that TitleStyle is supposed to look like:
![Doc screenshot](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/text/styles-images/builtinstyles.png)

### Basic Information

- Version with issue: 4.2.0.815419
- Last known good version: 4.0?
- IDE: VS 16.3.0 (Windows), VS 3.8 (build 1805) (Mac), Xcode 11
- Platform Target Frameworks:
  - iOS: 13.1

**Comment by samhouts** [comment]
@breyed  Can you please attach a small project that demonstrates this issue? Thanks!

**Comment by breyed** [comment]
@samhouts I don't have an isolated repro for this bug. It shows up in a production app that isn't doing anything special with its use of styles. I expect it will readily repro for you on an existing test project that makes use of `TitleStyle` on the latest XF and iOS. If you have trouble reproing, let me know.

**Comment by Adrien6** [comment]
Same here

[TitleStyleFont.zip](https://github.com/xamarin/Xamarin.Forms/files/3671480/TitleStyleFont.zip)

**Comment by PureWeen** [comment]
This looks to be specific to iOS 13 and not related to Xamarin.Forms

@jfversluis do you have any additional info on this? I know you've been researching things in this area

**Comment by jfversluis** [comment]
Yeah I've seen it happen to a couple of production apps I have on my phone. When I dig a bit deeper in the how and what it seems it's a bug/feature from Apple.

There is this [Radar report](https://openradar.appspot.com/6153065) that states that something has changed. From the linked site:

> Summary: When accessing the system font via UIFont.systemFont(ofSize:weight:), the font will often be rendered in Times New Roman instead of the expected SF font face. This occur for UITextView that render HTML as an attributed text.

Which is a bit strange regarding the HTML part. But this seems what is happening when we use the TitleStyles. I'm not entirely sure how these get resolved for iOS, but it seems something is going "wrong" there. The big question is: was this intended by Apple or not? I'm inclined to say we need to get this back to the regular built in font?

**Comment by breyed** [comment]
Should XF not be directly referencing the system font at all, but instead be calling [preferredFont(forTextStyle:)](https://developer.apple.com/documentation/uikit/uifont/1619030-preferredfont)?

**Comment by jfversluis** [comment]
That is probably right!
